src/xcode/ENA/ENA/Source/Services/TicketValidation/AccessToken/TicketValidationAccessTokenProcessor.swift

```diff
@@ -0,0 +1,85 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import ENASecurity
+import SwiftJWT
+
+struct TicketValidationAccessTokenProcessor {
+
+	// MARK: - Init
+
+	init(jwtVerification: JWTVerifying) {
+		self.jwtVerification = jwtVerification
+	}
+
+	// MARK: - Internal
+	
+	func process(
+		jwtWithHeadersModel: JWTWithHeadersModel,
+		accessTokenSignJwkSet: [JSONWebKey],
+		completion: @escaping (Result<TicketValidationAccessTokenResult, TicketValidationAccessTokenProcessingError>) -> Void
+	) {
+		guard let nonceBase64 = jwtWithHeadersModel.headers["x-nonce"] as? String else {
```

Comment:
Could this be resolved at the network or resource level.
It seems a little odd that the processing needs to resolve the header.